src/lib/API/api.js

```diff
@@ -384,6 +384,27 @@ export class APIService {
   async sendLoginVendorDetails(url, responseObject) {
     return this.sharedClient.post(url, responseObject)
   }
+
+  async getTokenTXs(token, address, fromBlock = null) {
+    const params = {
```

Comment:
refactored to use code from useRefund. params initializes as JS object (will be qs-stringified by axios)